lib/datadog/tracing/configuration/settings.rb

```diff
@@ -445,6 +445,34 @@ def self.extended(base)
                   )
                 end
               end
+
+              # Key-value map for explicitly re-mapping peer.service values
+              #
+              # @default `DD_TRACE_PEER_SERVICE_MAPPING` environment variable converted to hash
+              # @return [Hash]
+              option :peer_service_mapping do |o|
+                o.default do
+                  env_to_hash(
+                    Tracing::Configuration::Ext::SpanAttributeSchema::ENV_PEER_SERVICE_MAPPING,
+                    {},
+                    comma_separated_only: true
+                  )
+                end
+              end
```

Comment:
```suggestion
              option :peer_service_mapping do |o|
                o.env Tracing::Configuration::Ext::SpanAttributeSchema::ENV_PEER_SERVICE_MAPPING
                o.type :hash
                o.default({})
              end
```

After rebasing `master`, we can use the latest configuration options